Source/Core/DolphinQt/Debugger/MemoryViewWidget.cpp

```diff
@@ -191,18 +251,45 @@ void MemoryViewWidget::Update()
                                              QString{QChar::fromLatin1('.')};
       });
       break;
-    case Type::U16:
+    case Type::Hex16:
       update_values([&accessors](u32 address) {
         const u16 value = accessors->ReadU16(address);
         return QStringLiteral("%1").arg(value, 4, 16, QLatin1Char('0'));
       });
       break;
-    case Type::U32:
+    case Type::Hex32:
       update_values([&accessors](u32 address) {
         const u32 value = accessors->ReadU32(address);
         return QStringLiteral("%1").arg(value, 8, 16, QLatin1Char('0'));
       });
       break;
+    case Type::Unsigned8:
+      update_values(
+          [&accessors](u32 address) { return QString::number(accessors->ReadU8(address)); });
+      break;
+    case Type::Unsigned16:
+      update_values(
+          [&accessors](u32 address) { return QString::number(accessors->ReadU16(address)); });
+      break;
+    case Type::Unsigned32:
+      update_values(
+          [&accessors](u32 address) { return QString::number(accessors->ReadU32(address)); });
+      break;
+    case Type::Signed8:
+      update_values([&accessors](u32 address) {
+        return QString::number(Common::BitCast<signed char>(accessors->ReadU8(address)));
+      });
+      break;
+    case Type::Signed16:
+      update_values([&accessors](u32 address) {
+        return QString::number(Common::BitCast<signed short>(accessors->ReadU16(address)));
+      });
+      break;
+    case Type::Signed32:
+      update_values([&accessors](u32 address) {
+        return QString::number(Common::BitCast<signed int>(accessors->ReadU32(address)));
+      });
+      break;
```

Comment:
You can and should probably use `s8`/`s16`/`s32` here for the casts.

Reply:
Oh I didn't know you could use that. Thanks